Mathlib/GroupTheory/GroupAction/Primitive.lean

```diff
@@ -184,11 +184,11 @@ theorem isPreprimitive_of_surjective [IsPreprimitive M α] (hf : Function.Surjec
       exact isTrivialBlock_of_isBlock (IsBlock.preimage f hB) }
 
 @[to_additive]
-theorem isPreprimitive_congr (hφ : Function.Surjective φ) (hf : Function.Bijective f) :
+theorem IsPreprimitive.congr (hφ : Function.Surjective φ) (hf : Function.Bijective f) :
```

Comment:
This one was good as 
```suggestion
theorem isPreprimitive_congr (hφ : Function.Surjective φ) (hf : Function.Bijective f) :
```
since its conclusion is not `IsPreprimitive _ _` but `Iff _ _`